.github/workflows/ci.yml

```
@@ -90,13 +90,35 @@ jobs:
+      - name: Install pgvector and pg_trgm into template1 (postgres only)
         if: matrix.backend == 'postgres'
         run: |
           docker exec ${{ job.services.db.id }} psql -U yorishiro -d template1 \
             -c 'CREATE EXTENSION IF NOT EXISTS vector;' \
             -c 'CREATE EXTENSION IF NOT EXISTS pg_trgm;'
 
+      - name: Run migrations (postgres)
+        if: matrix.backend == 'postgres'
+        env:
+          DATABASE_URL: postgres://yorishiro:yorishiro@localhost:5432/yorishiro
+        run: DATABASE_URL='$(DATABASE_URL)' LOCO_ENV='$(LOCO_ENV)' cargo loco db migrate --locked
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:1:14: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)